expyfun/_eyelink_controller.py

```diff
@@ -137,14 +137,17 @@ class EyelinkController(object):
         Sample rate to use. Must be one of [250, 500, 1000, 2000].
     verbose : bool, str, int, or None
         If not None, override default verbose level (see expyfun.verbose).
+    calbration_keys : list
+        Keys that will trigger recalibration when check_recalibration.
 
     Notes
     -----
     The data will be saved to the ExperimentController ``output_dir``.
     If this was `None`, data will be saved to the current working dir.
     """
     @verbose_dec
-    def __init__(self, ec, link='default', fs=1000, verbose=None):
+    def __init__(self, ec, link='default', fs=1000, verbose=None,
+                 calibration_key=['c']):
```

Comment:
Do not use mutable default arguments. Tuple is immutable so `('c',)` is better here